internal/core/command/device.go

```diff
@@ -104,14 +109,16 @@ func executeCommandByDevice(
 	body string,
 	queryParams string,
 	isPutCommand bool,
-	lc logger.LoggingClient) (string, error) {
+	lc logger.LoggingClient,
+	r *http.Request,
```

Comment:
We talked previously about redundant fields and reducing the arguments to the functions which now accept `r *http.Request`. For example, you can derive the following from the `http.Request`:

1. `isPutCommand`
1. `queryParams`
1. `body`

Reply:
Yes, looks like I had implemented this (reduction of arguments to the said functions) in the corresponding `device-sdk-go` `PR` _only_, and will apply them here as well 👍